# thesis-2022
Supplementary thesis materials for each chapter that are too unwieldy to be included in thesis appendix are uploaded in their respective folders in this repository.

# Corrections and additions

## Corrections
### Chapter 4, preface
Supplementary materials can be found at https://github.com/rlrq/thesis-2022/tree/main/chapter%203 instead of the link provided in the thesis.

## Addition
### Chapter 3, Materials and Methods
3.4.8 GWAS (Section to be inserted immediately after 'F1 hybrid generation and characterisation')
GWAS was performed with easyGWAS (Grimm et al., 2017) using the 1001 Genomes Data (Alonso-Blanco et al., 2016) dataset and the TAIR10 gene annotation set. All other parameters were left as default. The .pheno file specifying pseudo-phenotypes based on PAV of 40A and/or 40B can be found at https://github.com/rlrq/thesis-2022/tree/main/chapter%203.

References for new section 3.4.8:
Alonso-Blanco, C., Andrade, J., Becker, C., Bemm, F., Bergelson, J., Borgwardt, K. M. M., Cao, J., Chae, E., Dezwaan, T. M. M., Ding, W., Ecker, J. R. R., Exposito-Alonso, M., Farlow, A., Fitz, J., Gan, X., Grimm, D. G. G., Hancock, A. M. M., Henz, S. R. R., Holm, S., . . . Zhou, X. (2016). “1,135 Genomes Reveal the Global Pattern of Polymorphism in Arabidopsis thaliana”. *Cell*, *166* (2), 481–491.
Grimm, D. G., Roqueiro, D., Salomé, P. A., Kleeberger, S., Greshake, B., Zhu, W., Liu, C., Lippert, C., Stegle, O., Schölkopf, B., Weigel, D., & Borgwardt, K. M. (2017). “Easygwas: A cloud-based platform for comparing the results of genome-wide association studies”. *Plant Cell*, *29*, 5–19.
